lib/src/neu/verifications/contract-mismatcher.ts

```diff
@@ -365,6 +452,25 @@ export class ContractMismatcher {
     }
   }
 
+  private findMismatchOnStringContent(
+    content: StringInput,
+    contractContentTypeToCheckWith: Type
+  ): Result<Mismatch[], Error> {
+    if (!content) {
+      return ok([]);
+    }
```

Comment:
why is lack of `content` automatically valid?

Reply:
hmm this has to be reworked then  i guess, i re-used the existing logic.